docs/deployment-aws.md

```diff
@@ -0,0 +1,202 @@
+# DCL Deployment on AWS using Terraform and Ansible
```

Comment:
Do we really need this doc if we have `running-node-aws-terraform-ansible/deployment.md`?